ScoutSuite/providers/azure/resources/base.py

```diff
@@ -3,11 +3,26 @@
 import abc
 import asyncio
 
-from ScoutSuite.providers.base.configs.resources import CompositeResources
+from ScoutSuite.providers.base.configs.resources import Resources, CompositeResources
 from ScoutSuite.providers.azure.facade.base import AzureFacade
 
 
-class AzureCompositeResources(CompositeResources, metaclass=abc.ABCMeta):
+class AzureResources(Resources, metaclass=abc.ABCMeta):
+
+    """This is the base class for Azure resources."""
+
+    def __init__(self, facade: AzureFacade, scope: dict = None):
+        """
+        :param scope: The scope holds the scope in which the resource is located. This usually means at least a region,
+                      but can also contain a VPC id, an owner id, etc. It should be used when fetching the data through
+                      the facade.
```

Comment:
Does this comment apply to Azure? I thought there weren't regions with Azure. I might very well be wrong.

Reply:
Pretty sure that's a copy paste error, I never saw Owner and VPCs in azure!

Reply:
Indeed, I updated the docstring. Nice catch. :)